projects/1.16.1/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2083 @@
+    "tetra/schematic/double/sickle/sickle.slot1": "镰刀材料",
+    "tetra.module.double/sickle.item_name": "镰刀",
+    "tetra.module.double/sickle.name": "镰刀",
+    "tetra.module.double/sickle.description": "一个轻微弯曲的镰刀，可以用来切割范围内的方块。\n \n和其他工具不同。它能够瞬间采掘方块，但仅限冷却完毕时。",
+    "tetra.module.double/sickle.material_name": "%s镰刀",
+    "tetra.module.double/sickle.prefix": "%s",
+    "tetra.slot.single/head": "头部",
+    "tetra.slot.single/handle": "手柄",
+    "tetra.slot.single/binding": "绑定",
+    "tetra.synergy.single/javelin": "投枪",
+    "tetra/schematic/single/handle/basic_handle.name": "基础手柄",
+    "tetra/schematic/single/handle/basic_handle.description": "用给定材料合成的基础手柄来替换手柄。\n \n基于材料提供完整度。材料§f密度§r略微减少§f攻击速度§r。",
+    "tetra/schematic/single/handle/basic_handle.slot1": "手柄材料",
+    "tetra.module.single/basic_handle.name": "基础手柄",
+    "tetra.module.single/basic_handle.description": "一个基础的手柄。手柄是完整度的主要来源。",
+    "tetra.module.single/basic_handle.material_name": "%s手柄",
+    "tetra/schematic/single/handle/light_handle.name": "轻手柄",
+    "tetra/schematic/single/handle/light_handle.description": "用给定材料合成的轻手柄来替换手柄。轻手柄使物品能够被投掷出去，但比其它手柄提供更少的完整度。\n \n基于材料提供完整度。材料§f密度§r增加§f投掷伤害§r，但略微减少§f攻击速度§r。",
+    "tetra/schematic/single/handle/light_handle.slot1": "手柄材料",
+    "tetra.module.single/light_handle.name": "轻手柄",
+    "tetra.module.single/light_handle.description": "轻手柄提供更少的完整度，但使物品能够被投掷出去。手柄是完整度的主要来源。",
+    "tetra.module.single/light_handle.material_name": "轻%s手柄",
+    "tetra.variant.light_handle/stick": "劣质轻手柄",
+    "tetra/schematic/single/handle/long_handle.name": "长手柄",
+    "tetra/schematic/single/handle/long_handle.description": "用给定材料合成的长手柄来替换手柄。长手柄提供更远的攻击范围。\n \n基于材料提供完整度。材料§f韧性§r和§f硬度§r增加§f使用范围§r。\n材料§f密度§r略微减少§f攻击速度§r。",
+    "tetra/schematic/single/handle/long_handle.slot1": "手柄材料",
+    "tetra.module.single/long_handle.name": "长手柄",
+    "tetra.module.single/long_handle.description": "长手柄提供更远的攻击范围。手柄是完整度的主要来源。",
+    "tetra.module.single/long_handle.material_name": " 长%s手柄",
+    "tetra.variant.long_handle/stick": "劣质长手柄",
+    "tetra.improvement.single_handle/wrap_string.name": "绳裹柄",
+    "tetra.improvement.single_handle/wrap_string.description": "轻微增加攻击和攻速，减少手柄负面效果。",
+    "tetra.improvement.single_handle/wrap_leather.name": "皮革裹柄",
+    "tetra.improvement.single_handle/wrap_leather.description": "轻微增加攻击和攻速，减少手柄负面效果。",
+    "tetra.improvement.single_handle/wrap_hide.name": "兔皮裹柄",
+    "tetra.improvement.single_handle/wrap_hide.description": "轻微增加攻击和攻速，减少手柄负面效果。",
+    "tetra.improvement.single_handle/wrap_vine.name": "藤蔓裹柄",
+    "tetra.improvement.single_handle/wrap_vine.description": "轻微增加攻击和攻速，减少手柄负面效果。",
+    "tetra/schematic/single/handle/wrap_handle.name": "裹柄",
+    "tetra/schematic/single/handle/wrap_handle.slot1": "材料",
+    "tetra/schematic/single/handle/wrap_handle.description": "使用给定材料裹柄。轻微增加攻击和攻速，并减少手柄可能导致的负面效果。",
```

|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
    "tetra/schematic/single/handle/wrap_handle.description": "使用给定材料裹柄。较小地增加伤害和挥动速度，并且减少手柄可能导致的负面效果。",
```
同 L582 和 L811